static/src/javascripts/projects/common/modules/identity/modules/show-errors.js

```diff
@@ -6,7 +6,7 @@ const formErrorHolderClassName = 'js-errorHolder';
 
 const genericErrorMessage = 'Sorry, something went wrong';
 
-export const push = (error: Error, action: string = 'none'): Promise<void> =>
+export const push = (error: any, action: string = 'none'): Promise<void> =>
```

Comment:
this is on purpose as there's no real guarantee show-errors will get an error at runtime as it interacts with a lot of non-flow code so i'd rather have it working with any type